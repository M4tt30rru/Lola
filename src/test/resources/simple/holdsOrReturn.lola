##Import "std.lola"
##Find ##Expression(_exp) || return ##Expression(_def);
 ##replace if (!(##(_exp))) return ##(_def); else ;

f() != null || return "f returned null!";
