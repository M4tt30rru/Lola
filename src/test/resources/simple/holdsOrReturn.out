if (!(f() != null))
 return "f returned null!";
else
 ;
